### DISCUSSION TITLE
k5.4编译失败

### DISCUSSION THREAD

**Discussion by jh0072110** [immortalwrt/immortalwrt#521] [Q&A] [answered]
make[2]: *** [package/Makefile:70: package/install] Error 255
4628
make[1]: *** [package/Makefile:111: /workdir/openwrt/staging_dir/target-x86_64_musl/stamp/.package_install] Error 2
4629
make: *** [/workdir/openwrt/include/toplevel.mk:230: world] Error 2
4630
make[2]: Leaving directory '/workdir/openwrt'
4631
make[1]: Leaving directory '/workdir/openwrt'
4632
Error: Process completed with exit code 2.
以上为报错日志部分…感谢付出

**Comment by 1715173329** [comment]
请问您的日志是按长度收费吗？

**Comment by jh0072110** [comment]
> 请问您的日志是按长度收费吗？

不好意思啊早上拿手机截了一段，我去电脑上导出完整log啊……

**Comment by jh0072110** [comment]
> 请问您的日志是按长度收费吗？
[11_Compile the firmware.txt](https://github.com/immortalwrt/immortalwrt/files/7632145/11_Compile.the.firmware.txt)

**Comment by 1715173329** [comment, marked as answer]
取消勾选 kmod-fs-exfat。
